code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -1652,13 +1652,14 @@
 	search_metadata = "boris"
 
 /datum/design/borg_upgrade_rped
-	name = "Rapid Part Exchange Device"
+	name = "Rapid Part Exchange Device Expanded"
 	id = "borg_upgrade_rped"
 	build_type = MECHFAB
 	build_path = /obj/item/borg/upgrade/rped
 	materials = list(
-		/datum/material/iron =SHEET_MATERIAL_AMOUNT*5,
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT*7.5,
 		/datum/material/glass =SHEET_MATERIAL_AMOUNT * 2.5,
+		/datum/material/silver = HALF_SHEET_MATERIAL_AMOUNT*2.5
```

Comment:
spacing around *s pls (or failing that do me a favor and make this list self consistent, that's most of my concern in this case)